commit 566198730352f2353fd8332794409d7c9edf80e2
Author: Louis Sautier <sautier.louis@gmail.com>
Date:   Wed Sep 8 00:53:28 2021 +0200

    tests: skip man and python2
    
    Those tests fail respectively when the documentation isn't built and when
    python2 isn't available (probably specific to Gentoo).

diff --git a/tests/test_formatters/test_py.py b/tests/test_formatters/test_py.py
index 92b17604..28d2dd42 100644
--- a/tests/test_formatters/test_py.py
+++ b/tests/test_formatters/test_py.py
@@ -16,7 +16,7 @@ def _check_interpreter(interpreter):
         return False
 
 
-@parameterized(["python2", "python3"])
+@parameterized(["python3"])
 @with_setup(usual_setup_func, usual_teardown_func)
 def test_paranoia(interpreter):
     if not _check_interpreter(interpreter):
diff --git a/tests/test_options/test_help.py b/tests/test_options/test_help.py
index d934724a..60bc40a0 100644
--- a/tests/test_options/test_help.py
+++ b/tests/test_options/test_help.py
@@ -17,7 +17,7 @@ def test_help():
 
 
 @with_setup(usual_setup_func, usual_teardown_func)
-def test_man():
+def _test_man():
     yelp = subprocess.check_output(
         ['./rmlint', '--show-man'], stderr=subprocess.STDOUT
     ).decode('utf-8')
